nano/templates/sleeper.tmpl

```diff
@@ -0,0 +1,154 @@
+{{else}}
+	<h2>Sleeper Console</h2>
+	<div class="item">
+		<div class="itemLabel">
+			<b>Scanner Module</b>:
+		</div>
+		<div class="itemContent">
+			{{if data.occupant}}
+				<div class="good">active</div>
+			{{else}}
+				<div class="bad">unoccupied</div>
+			{{/if}}
+		</div>
+	</div>
+	<div class="item">
+		<div class="itemLabel">
+			<b>Controls</b>:
+		</div>
+		<div class="itemContent">
+			{{:helper.link('Eject occupant', 'eject', {'ejectify' : 1}, (data.occupant) ? null : 'disabled')}}
+			{{:helper.link('Print report', 'print', {'print' : 1}, (data.occupant) ? null : 'disabled')}}
+		</div>
+	</div>
+	{{if data.occupant}}
+		<br>
+		<h3>Scanner Data</h3>
+		<div class="item">
+			<div class="itemLabelWide">
+				<b>Mental activity</b>:
+			</div>
+			<div class="itemContentMedium">
+				{{if data.occupant.stat.string == 'Dead'}}
+					<span class="bad">
+				{{else data.occupant.stat.string == 'Unconscious'}}
+					<span class="average">
+				{{else}}
+					<span>
+				{{/if}}
+				{{:data.occupant.stat.string}}
+				</span>
+			</div>
+		</div>
+		<div class="item">
+			<div class="itemLabelWide">
+				<b>Health rating</b>:
+			</div>
+			<div class="itemContentMedium">
+				{{:helper.displayBar(data.occupant.health.combined, 0, 100, (data.occupant.health.combined < 50) ? 'bad' : (data.occupant.health.combined < 90) ? 'average' : 'good', Math.round(data.occupant.health.combined)+'%')}}
+			</div>
+			<br>
+			<div class="itemLabelWide">
+				Brute damage:
+			</div>
+			<div class="itemContentMedium">
+				{{:helper.displayBar(data.occupant.health.brute, 0, 100, (data.occupant.health.brute > 20) ? 'bad' : 'average', Math.round(data.occupant.health.brute)+'%')}}
+			</div>
+			<br>
+			<div class="itemLabelWide">
+				Respiratory damage:
+			</div>
+			<div class="itemContentMedium">
+				{{:helper.displayBar(data.occupant.health.oxy, 0, 100, (data.occupant.health.oxy > 10) ? 'bad' : 'average', Math.round(data.occupant.health.oxy)+'%')}}
+			</div>
+			<br>
+			<div class="itemLabelWide">
+				Toxin content:
+			</div>
+			<div class="itemContentMedium">
+				{{:helper.displayBar(data.occupant.health.toxin, 0, 100, (data.occupant.health.toxin > 5) ? 'bad' : 'average', Math.round(data.occupant.health.toxin)+'%')}}
+			</div>
+			<br>
+			<div class="itemLabelWide">
+				Burn severity:
+			</div>
+			<div class="itemContentMedium">
+				{{:helper.displayBar(data.occupant.health.burn, 0, 100, (data.occupant.health.burn > 10) ? 'bad' : 'average', Math.round(data.occupant.health.burn)+'%')}}
+			</div>
+		</div>
+		{{if data.occupant.health.pulse}}
+		<div class="item">
+			<div class="itemLabelWide">
+				<b>Pulse</b>:
+			</div>
+			<div class="itemContentMedium">
+				<b>{{:data.occupant.health.pulse.value}}</b>bpm ({{:data.occupant.health.pulse.state}})
+			</div>
+		</div>
+		{{/if}}
+		<!--{{if data.occupant.health.paralysis}}
```

Comment:
Yes. I cross-referenced my template with that of the current Baystation UI, and it is missing there, too. In fact, all SetParalysis() calls are setting the /mob/living/paralysis variable to 0.

I left the comment in there, because the old UI _did_ show this value, and I wanted to make as few changes to game-affecting functionality, as possible ("fixing" it would only require uncommenting this block).

I removed the commented block in https://github.com/Aurorastation/Aurora.3/pull/1043/commits/e744f714371973a1ec5357c0d0d2a47467f68dea .